meta_tests/conftest_test.py

```diff
@@ -44,30 +55,105 @@ def setup_method(self, method):
         self.no_offheap_memtables_test.get_closest_marker.side_effect = _mock_responses({"no_offheap_memtables": True})
         self.depends_driver_test.get_closest_marker.side_effect = _mock_responses({"depends_driver": True})
 
-    def test_regular_test(self):
-        assert not _is_skippable(item=self.regular_test)
-        assert _is_skippable(item=self.regular_test, include_non_upgrade_tests=False)
-        assert _is_skippable(item=self.regular_test, include_non_resource_intensive_tests=False)
+    @pytest.mark.parametrize("item", [upgrade_test, resource_intensive_test, vnodes_test, depends_driver_test])
+    @pytest.mark.parametrize("a_class_marked", [no_patch, upgrade_test_patch, resource_intensive_patch, vnodes_patch,
+                                           no_vnodes_patch, no_offheap_memtables_patch, depends_driver_patch])
+    def test_skip_if_no_config(self, item, a_class_marked):
+        dtest_config = DTestConfigMock()
+        with patch('conftest.any_class_in_module_has_mark', a_class_marked):
+            assert is_skippable(item, dtest_config, False)
```

Comment:
I would use param name before `False` here to make it more readable.
up to you.
```
assert is_skippable(item, dtest_config, skip_marked=False)
```